modules/tinymce/src/plugins/lists/test/ts/browser/BackspaceDeleteFromBlockIntoLiTest.ts

```diff
@@ -13,6 +13,27 @@ describe('Browser Test: .RemoveTrailingBlockquoteTest', () => {
     base_url: '/project/tinymce/js/tinymce'
   }, [ Plugin ]);
 
+  it('TINY-8592: backspace from p outside of table with no change', () => {
```

Comment:
Since there is no test failing for the incorrect compare more tests are needed here.

- [ ] Add tests that covers delete from within a the same cell, from cell to text from text to cell from cell to other cell and from text to text.